contracts/proxy/receivers.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "@openzeppelin/contracts/proxy/TransparentUpgradeableProxy.sol";
+
+contract InstaAaveV2MigratorReceiver is TransparentUpgradeableProxy {
+    constructor(address _logic, address admin_, bytes memory _data) public TransparentUpgradeableProxy(_logic, admin_, _data) {}
```

Comment:
So what is `bytes memory _data`?

Reply:
After setting the implementation, if we need to set anything on the implementation contract like variables.... For that function calldata